.travis.yml

```diff
@@ -122,6 +122,9 @@ before_script:
   - tar xzf tar-output/ledgersmb-travis.tar.gz
   - cd ledgersmb
   - cp conf/ledgersmb.conf.default ledgersmb.conf
```

Comment:
Rather than copying the config file, we should simply set the LSMB_CONFIG_FILE environment variable